File: music-domain/src/main/java/com/music/sale/domain/product/ProductImage.java
package com.music.sale.domain.product;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
@AllArgsConstructor
public class ProductImage {
    private final Long id;
    private final Long productId;
    private final String url;
    private final boolean isThumbnail;
    private final int imageOrder;
    private final long fileSize;
    private final String fileName;
    private final String fileType;
}

File: music-domain/src/main/kotlin/com/music/sale/domain/product/ProductImage.kt
package com.music.sale.domain.product

data class ProductImage(
    val id: Long? = null,
    // 어떤 Product 인스턴스에 속하는지
    val productId: Long,
    val imageUrl: String,
    var isThumbnail: Boolean = false,
    val order: Int = 0,
    val altText: String? = null,
)
